src/NServiceBus.AcceptanceTests/Mutators/Issue_1980.cs

```diff
@@ -8,18 +8,17 @@
 
     public class Issue_1980 : NServiceBusAcceptanceTest
     {
+        static Context testContext = new Context();
```

Comment:
it is a single instance of a singe type that is sued once. no need for injection. curious as to why this wont work with async?